easy-table-of-contents.php

```diff
@@ -424,6 +424,11 @@ public static function is_eligible() {
 			if ( empty( $post ) || ! $post instanceof WP_Post ) {
 				return FALSE;
 			}
+			
+			global $wp_current_filter;
+			if ( $wp_current_filter[0] === 'wp_head' ) {
+				return false;
+			}
```

Comment:
Could you please provide a test case where this works?

I added this snippet to a live site that uses Yoast for open graph. The description meta still showed signs of ezTOC being run. So, fome it did not work.

Reply:
I implement this in various websites with yoast and genesis, this is probably explain why doesn't work with theme that are not genesis based.
Maybe the filter that execute is different but a bit of debugging can help to define what is.

Reply:
ok, I could not see any issue with Yoast, since this is a harmless addition I commited it in 
b6877c0. I also added `get_the_excerpt` as a filter to exclude too.

Reply:
I was finally able to replicate an issue with Yoast even with this change. My best guess an infinite loop was occurring when applying `the_content` filter. No matter how much I increased the memory, I would still get an out of memory error.

However, if I moved this code to the top of `the _content()` method from the `is_eligible()` method, it worked just fine.

Are you certain you were adding it to the  `is_eligible()` method?

I'm going to add a little helper method and move this code to it and add a call to it from the `the _content()` method instead.

Do you think that will work for the issue you encountered too?

Reply:
I can do some tests maybe before the end of the next week.